development/skinning.md

```diff
@@ -0,0 +1,182 @@
+ | | 
+ `--alpheios-compact-panel-icon-color` | `--alpheios-icon-color` | Panel header icons color
+ `--alpheios-compact-panel-icon-bg` | `--alpheios-toolbar-bg-color` |  Panel header icons background color
+ `--alpheios-compact-panel-icon-color-hover` | `--alpheios-icon-color-hover` | Panel header icons color - hover state
+ `--alpheios-compact-panel-icon-bg-hover` | `--alpheios-toolbar-bg-color` | Panel header icons background color - hover state
+ `--alpheios-compact-panel-icon-color-active` | `--alpheios-icon-color-active` | Panel header icons color - active state
+ Lookup |  | 
+ `--alpheios-lookup-border-color` | `--alpheios-border-color` | Lookup panel border color (in toolbar)
+ `--alpheios-lookup-bg` |  `--alpheios-text-bg-color` | Lookup panel panel background (in toolbar)
+ `--alpheios-lookup-input-border-color` | `--alpheios-border-color` | Lookup input border color
+ `--alpheios-lookup-button-color` | `--alpheios-btn-primary-font-color` | Lookup main button - font color
+ `--alpheios-lookup-button-bg` | `--alpheios-btn-primary-bg-color` | Lookup main button - background color
+ `--alpheios-lookup-button-border-color` | `--alpheios-btn-primary-border-color` | Lookup main button - border color
+ `--alpheios-lookup-button-color-hover` | `--alpheios-btn-primary-font-color-hover` | Lookup main button - font color - hover state
+ `--alpheios-lookup-button-bg-hover` | `--alpheios-btn-primary-bg-color-hover` | Lookup main button - background color - hover state
+ `--alpheios-lookup-button-border-color-hover` | `--alpheios-btn-primary-border-color-hover` | Lookup main button - border color - hover state
+ Inflections Browser |  | 
+ `--alpheios-inflect-browser-title-color` | `--alpheios-link-color-on-light` | Inflections Browser - main title - font color
+ `--alpheios-inflect-button-control-color` | `--alpheios-btn-secondary-font-color` | Inflections Browser - expand/collapse button - font color
+ `--alpheios-inflect-button-control-bg` | `--alpheios-btn-secondary-bg-color` | Inflections Browser - expand/collapse button - background color
+ `--alpheios-inflect-button-control-border-color` | `--alpheios-btn-secondary-border-color`  | Inflections Browser - expand/collapse button - border color
+ `--alpheios-inflect-button-control-color-hover` | `--alpheios-btn-secondary-font-color-hover`  | Inflections Browser - expand/collapse button - font color - hover state
+ `--alpheios-inflect-button-control-bg-hover` | `--alpheios-btn-secondary-bg-color-hover`  | Inflections Browser - expand/collapse button - background color - hover state
+ `--alpheios-inflect-button-control-border-color-hover` | `--alpheios-btn-secondary-border-color-hover`  | Inflections Browser - expand/collapse button - border color - hover state
+ `--alpheios-inflect-match-cell-border-color` | `--alpheios-highlight-border-color` | Inflections Browser - border color for cells that has match

```

Comment:
Maybe it's better to indicate in a description that this is for cells that has a full feature match (if I remember correctly). Also, maybe it's better to rename it to something like `--alpheios-inflect-feat-match-cell-border-color` to signify that too?

Reply:
agree

Reply:
no, you are not right - it is not for full match, it is for morphologyMatch

`infl-cell--morph-match`: `cell.morphologyMatch`

I will add morphology to description

Reply:
Great, thanks!